app/components/Header.tsx

```diff
@@ -100,9 +101,43 @@ export function Header() {
               >
                 Circulars
               </NavLink>,
-              <NavLink className="usa-nav__link" to="/docs" key="/docs">
-                Documentation
-              </NavLink>,
+              useFeature('SCHEMA') ? (
+                <>
+                  <NavDropDownButton
+                    to="/docs"
+                    type="button"
+                    key="docs"
+                    label="Documentation"
+                    isOpen={docsMenuIsOpen}
+                    onToggle={() => setDocsMenuIsOpen(!docsMenuIsOpen)}
+                    menuId="notices"
+                  />
+                  <Menu
+                    id="docs"
+                    isOpen={docsMenuIsOpen}
+                    items={[
+                      <Link
+                        key="docs"
+                        to="/docs"
+                        onClick={() => setDocsMenuIsOpen(!docsMenuIsOpen)}
+                      >
+                        Documentation
+                      </Link>,
+                      <Link
+                        key="schema"
+                        to="/schema-browser"
+                        onClick={() => setDocsMenuIsOpen(!docsMenuIsOpen)}
+                      >
+                        Schema Browser
+                      </Link>,
+                    ]}
+                  />
+                </>
+              ) : (
+                <NavLink className="usa-nav__link" to="/docs" key="/docs">
+                  Documentation
+                </NavLink>
+              ),
```

Comment:
I'd rather that the Documentation nav button not be a dropdown right now.

The user nav button is a dropdown, but the dropdown menu items exactly match the sidenav items on the /user route. And it's a small number of menu items.

On the other hand, the Documentation section has a large (and likely to grow larger still) number of menu items.

Reply:
Well... on the other hand, there will be a schema for circulars, which are not notices.

OK, I guess I'm alright with putting it under /notices.

Reply:
Ah thats a good point, I forgot about the circulars ones. That does make it seem like it would be better under docs. That makes a lot more sense now, I will update it to be under docs and add update the other nav stuff

Reply:
Small update, the docs header nav is no longer a dropdown, and i added a side nav option to get to the schema-browser and a back button. I did not move the files though. In trying to move /routes/schema-browser to /routes/docs/schema it broke, the side nav. It only shows the doc level side nav